index.js

```diff
@@ -1,6 +1,6 @@
-const pprops = require('p-props');
+const pMap = require('p-map');
```

Comment:
Why the switch?

Reply:
My reasoning was sort of as follows, in no particular order.
1. `p-props` uses `p-map` underneath so eliminates a layer of libraries.
2. `p-map` is a cleaner syntax for using a _mapping_ function (in my opinion).
3. In order for the concurrency option to work, the promise must be in the `mapper` function.
4. Using `p-map` means I can loop over the array elements natively. In order to use `p-props` I would first have to loop over the array to build the object, then use `p-props` with a mapping function to fill in the values.

In the current version, I found that we were essentially creating an object that was built using all promises similar to this:
```
const obj = {
  val: promFunc(),
  val2: promFunc(),
  val3: promFunc(),
  ...
  val22000: promFunc()
};
```
We then used `p-props` to simply ensure all the promises had resolved *however* they all get fired at the exact same time by the nature of the way the function was built. The `p-map` library will only limit the number of concurrent promises running if it has control over when to fire them via the mapping function.